### PR TITLE
fix test cases of compute resources

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**

### CODE REVIEW

**Review by Jason-Zhang9309 — COMMENTED** [review]
self review finished

File: huaweicloud/resource_huaweicloud_compute_volume_attach_test.go [hunk 1]
```diff
@@ -144,77 +148,143 @@ func testAccCheckComputeV2VolumeAttachDevice(
...
-  volume_id = "${huaweicloud_evs_volume.test.id}"
+  instance_id = huaweicloud_compute_instance.instance_1.id
+  volume_id = huaweicloud_evs_volume.test.id
   device = "/dev/vdb"
 }
-`, HW_AVAILABILITY_ZONE, HW_AVAILABILITY_ZONE, HW_NETWORK_ID)
+`, rName, rName)
+}
+
+func testAccComputeV2VolumeAttach_timeout(rName string) string {
```
Comment (niuzhenguo) [inline huaweicloud/resource_huaweicloud_compute_volume_attach_test.go]: just get rid of this timeout test.

Comment (Jason-Zhang9309) [reply huaweicloud/resource_huaweicloud_compute_volume_attach_test.go]: fixed

**Comment by ShiChangkuo** [comment]
@Jason-Zhang9309 it's very appreciated if the testing resuts was attached in the PR

**Comment by Jason-Zhang9309** [comment]
make testacc TEST='./huaweicloud' TESTARGS='-run TestAccComputeV2VolumeAttach_device'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run TestAccComputeV2VolumeAttach_device -timeout 360m -parallel 4
=== RUN   TestAccComputeV2VolumeAttach_device
=== PAUSE TestAccComputeV2VolumeAttach_device
=== CONT  TestAccComputeV2VolumeAttach_device
--- PASS: TestAccComputeV2VolumeAttach_device (236.09s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       236.135s

make testacc TEST='./huaweicloud' TESTARGS='-run TestAccComputeV2ServerGroup_affinity'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run TestAccComputeV2ServerGroup_affinity -timeout 360m -parallel 4
=== RUN   TestAccComputeV2ServerGroup_affinity
=== PAUSE TestAccComputeV2ServerGroup_affinity
=== CONT  TestAccComputeV2ServerGroup_affinity
--- PASS: TestAccComputeV2ServerGroup_affinity (154.86s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       154.933s

make testacc TEST='./huaweicloud' TESTARGS='-run TestAccComputeV2InterfaceAttach_Basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run TestAccComputeV2InterfaceAttach_Basic -timeout 360m -parallel 4
=== RUN   TestAccComputeV2InterfaceAttach_Basic
=== PAUSE TestAccComputeV2InterfaceAttach_Basic
=== CONT  TestAccComputeV2InterfaceAttach_Basic
--- PASS: TestAccComputeV2InterfaceAttach_Basic (217.39s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       217.429s

**Pull request merged**